blocks/library/columns/index.js

```diff
@@ -54,34 +52,20 @@ export const settings = {
 			type: 'number',
 			default: 2,
 		},
-		align: {
-			type: 'string',
-		},
 	},
 
-	description: __( 'A multi-column layout of content.' ),
-
-	getEditWrapperProps( attributes ) {
-		const { align } = attributes;
-
-		return { 'data-align': align };
+	supports: {
+		align: [ 'wide', 'full' ],
 	},
 
-	edit( { attributes, setAttributes, className, focus } ) {
-		const { align, columns } = attributes;
+	description: __( 'A multi-column layout of content.' ),
+
+	edit( { attributes, setAttributes, className, isSelected } ) {
+		const { columns } = attributes;
 		const classes = classnames( className, `has-${ columns }-columns` );
 
 		return [
-			...focus ? [
-				<BlockControls key="controls">
-					<BlockAlignmentToolbar
-						controls={ [ 'wide', 'full' ] }
-						value={ align }
-						onChange={ ( nextAlign ) => {
-							setAttributes( { align: nextAlign } );
-						} }
-					/>
-				</BlockControls>,
+			isSelected ? [
```

Comment:
Since there's only a single entry, this could be simplified to:

```js
isSelected && (
```

Also dropping the `key` from `InspectorControls`